docs/src/visualization.md

```diff
@@ -0,0 +1,185 @@
+# Visualization
+
+## Plots.jl
+
+!!! note "Add Plots.jl to your Project.toml"
+    To enable plotting by Plots.jl, use `using Plots`.
+
+Plotting using Plots.jl supports only plotting distributions. See the example
+below.
+
+```@example
+using EnsembleKalmanProcesses.ParameterDistributions
+prior_u1 = constrained_gaussian("positive_with_mean_2", 2, 1, 0, Inf)
+prior_u2 = constrained_gaussian("four_with_spread_5", 0, 5, -Inf, Inf, repeats=4)
+prior = combine_distributions([prior_u1, prior_u2])
+
+using Plots
+p = plot(prior)
+```
+
+## Makie.jl
+
+!!! note "Add a Makie-backend package to your Project.toml"
+    Import one of the Makie backends (GLMakie, CairoMakie, WGLMakie, RPRMakie,
+    etc.) to enable these functions!
+
+Plotting functionality is provided by Makie.jl through a package extension. See
+the [documentation](@ref Visualize) for a list of all the available plotting
+functions. The plotting functions have the same signature that one would expect
+from Makie plotting functions. See the
+[plot methods section](https://docs.makie.org/dev/explanations/plot_method_signatures)
+in Makie documentation for more information.
+
+```@setup makie_plots
+# Fix random seed, so plots don't change
+import Random
+rng_seed = 1234
+rng = Random.MersenneTwister(rng_seed)
+
+using LinearAlgebra
+using EnsembleKalmanProcesses
+using EnsembleKalmanProcesses.ParameterDistributions
+
+G(u) =
+    [1.0 / abs(u[1]), sum(u[2:3]), u[3], u[1]^2 - u[2] - u[3], u[1], 5.0] .+ 0.1 * randn(6)
+true_u = [3, 1, 2]
+y = G(true_u)
+Γ = (0.1)^2 * I
+
+prior_u1 = constrained_gaussian("positive_with_mean_1", 2, 1, 0, Inf)
+prior_u2 = constrained_gaussian("four_with_spread_2", 0, 5, -Inf, Inf, repeats = 2)
```

Comment:
Rename: "two_with_spread_2", (the "four" in the above example refers to the fact it is a 4-D distribution).